Kraken historical price tests fail too often

Needs to be made more robust without really slowing down the test suite. If it continues to fail as often as it does, it just slows down the work / and debugging CI runs in other PRs.

```  ------------------------------ Captured log call -------------------------------
  ERROR    telliot_feed_examples.sources.price.historical.kraken:kraken.py:134 {'error': "<class 'requests.exceptions.ReadTimeout'>", 'exception': ReadTimeout(ReadTimeoutError("HTTPSConnectionPool(host='api.kraken.com', port=443): Read timed out. (read timeout=0.5)"))}
  ____________________________ test_kraken_get_trades ____________________________
  
      @pytest.mark.asyncio
      async def test_kraken_get_trades():
          """Retrieve all price data given a timestamp and surrounding time period."""
          six_hours = 60 * 60 * 6  # seconds
          trades, t = await KrakenHistoricalPriceService().get_trades(
              "eth",
              "usd",
              period=six_hours,
              ts=1647782323,
          )
      
  >       assert isinstance(t, datetime)
  E       assert False
  E        +  where False = isinstance(None, datetime)
  
  ```
  [link](https://github.com/tellor-io/telliot-feed-examples/runs/6453547624?check_suite_focus=true)

--- a/src/telliot_feeds/sources/price/historical/poloniex.py
+++ b/src/telliot_feeds/sources/price/historical/poloniex.py
@@ -96,17 +96,17 @@ class PoloniexHistoricalPriceService:
         d = self.get_url(request_url)
         trades = []
 
-        if "error" in d:
+        if "error" in str(d):
             logger.error(d)
             return None, None
 
         elif "response" in d:
             rsp = d["response"]
-            if type(rsp) == dict and "error" in rsp:
-                logger.error(f'Error in poloniex response: {rsp["error"]}')
+            if isinstance(rsp, list):
+                trades = rsp
+            else:
+                logger.error(f"Unexpected response type: {type(rsp)}")
                 return None, None
-            trades = rsp
-
         else:
             raise Exception("Invalid response from get_url")
 

--- a/tests/sources/test_historical_price_sources.py
+++ b/tests/sources/test_historical_price_sources.py
@@ -38,7 +38,6 @@ def validate_price(v, t):
     print(t)
 
 
-@pytest.mark.skip("failing when run with unrelated tests")
 @pytest.mark.asyncio
 async def test_kraken_get_price():
     """Retrieve singular price close to given timestamp."""
@@ -49,7 +48,6 @@ async def test_kraken_get_price():
     validate_price(v, t)
 
 
-@pytest.mark.skip("failing when run with unrelated tests")
 @pytest.mark.asyncio
 async def test_kraken_get_trades():
     """Retrieve all price data given a timestamp and surrounding time period."""
@@ -84,15 +82,19 @@ async def test_kraken_get_trades():
 @pytest.mark.asyncio
 async def test_poloniex_get_price():
     """Retrieve single historical price close to given timestamp."""
-    v, t = await PoloniexHistoricalPriceService().get_price("eth", "dai", ts=1645813159)
+    six_hours = 60 * 60 * 6  # seconds
+    v, t = await PoloniexHistoricalPriceService().get_price("eth", "tusd", period=six_hours, ts=1647782323)
     validate_price(v, t)
 
-    v, t = await PoloniexHistoricalPriceService().get_price("eth", "tusd", ts=1645822159)
+    # Returns {"code": 500, "message": "System error"} for dai...
+    # v, t = await PoloniexHistoricalPriceService().get_price("eth", "dai", ts=1645822159)
+    # validate_price(v, t)
 
-    v, t = await PoloniexHistoricalPriceService().get_price("btc", "dai", ts=1645813159)
-    validate_price(v, t)
+    # v, t = await PoloniexHistoricalPriceService().get_price("btc", "dai", ts=1645813159)
+    # validate_price(v, t)
 
     v, t = await PoloniexHistoricalPriceService().get_price("btc", "tusd", ts=1645822159)
+    validate_price(v, t)
 
 
 @pytest.mark.asyncio

